src/config.rs

```diff
@@ -153,6 +160,33 @@ pub fn get_config<'a>(
         no_style: style::get_no_style(),
         max_buffered_lines: 32,
         paging_mode,
+        show_line_numbers: opt.show_line_numbers,
+        number_minus_foreground_color: get_color_option_or_default(
+            opt.number_minus_foreground_color.as_ref(),
+            &opt.hunk_color,
+        ),
+        number_plus_foreground_color: get_color_option_or_default(
+            opt.number_plus_foreground_color.as_ref(),
+            &opt.hunk_color,
+        ),
+        number_divider_right_foreground_color: get_color_option_or_default(
+            opt.number_divider_right_foreground_color.as_ref(),
+            &opt.hunk_color,
+        ),
+        number_divider_center_foreground_color: get_color_option_or_default(
+            opt.number_divider_center_foreground_color.as_ref(),
+            &opt.hunk_color,
+        ),
+        number_divider_center_string: opt.number_divider_center_string.as_ref(),
+        number_divider_right_string: opt.number_divider_right_string.as_ref(),
+    }
+}
+
+fn get_color_option_or_default(color_option: Option<&String>, default: &String) -> Color {
+    let _number_color = color_option.clone();
+    match color_option {
+        Some(x) => return color_from_rgb_or_ansi_code(&x),
+        None => return color_from_rgb_or_ansi_code(default),
```

Comment:
Can we use `color_from_rgb_or_ansi_code_with_default()` here?

Reply:
Yep!! Updated.